speech_recognition_msgs/msg/WordInfo.msg

```diff
@@ -0,0 +1,7 @@
+# please see below URL for more details.
+# https://cloud.google.com/speech-to-text/docs/reference/rest/v1/speech/recognize#wordinfo
+float64 start_time
+float64 end_time
```

Comment:
`startTime` and `endTime` of original WordInfo are Duration formatted string. I think `duration` is more compatible to it than `float64`.